examples/optional-components/app.config

```diff
@@ -0,0 +1,8 @@
+EXTERNAL_REGISTRY_BASE_DOMAIN=quay.io/example-org
+REPOSITORY_NAME='example'
+DOCKERTAG=quay.io/example-org/example
+
```

Comment:
Can you add a check in here for `$NAMESPACE`? I'd like to prevent someone copying this from running into a problem where `ROK8S_ENABLE_OPTIONAL_CONFIGS` is `yes` but `NAMESPACE` isn't defined where this would lead to "successful" deploy with 0 files deployed.